### PR TITLE
[l10n] Update Italian translation

### PR DESCRIPTION
**Pull request by MiloCasagrande**

### CODE REVIEW

File: po/it.po [hunk 1]
```diff
@@ -88,60 +70,66 @@ msgstr ""
...
 
-#: src/screenshotdialog.c:96
+#: ../src/screenshotdialog.c:96
 msgid "Share this screenshot with the requesting application?"
 msgstr "Condividere questa schermata con l'applicazione che la richiede?"
 
-#: src/screenshotdialog.ui:10
-msgid "Screenshot"
-msgstr "Schermata"
+#~ msgid "Share Details"
```
Comment (mariospr) [inline po/it.po]: This doesn't seem right: you're **removing** this translation, not updating it. Same for all the other strings below (`Share`, "Make changes before..."...).

Comment (MiloCasagrande) [reply po/it.po]: Hmmm, sorry... I didn't notice that.
What is the correct Make command to extract the translations and update the base file?

Comment (mariospr) [reply po/it.po]: Sorry, I did miss this message. I think you're looking for `make -C po/ update-po`

**Comment by MiloCasagrande** [comment]
Wow, forgot to update this PR a while back.
It should be fixed and really updated now.

**Comment by mariospr** [comment]
It looks good now, thanks

**Pull request merged**